engine/src/main/java/org/terasology/rendering/logic/FloatingTextRenderer.java

```diff
@@ -43,13 +43,7 @@
 import java.util.Arrays;
 import java.util.Map;
 
-import static org.lwjgl.opengl.GL11.GL_DEPTH_TEST;
-import static org.lwjgl.opengl.GL11.glDisable;
-import static org.lwjgl.opengl.GL11.glEnable;
-import static org.lwjgl.opengl.GL11.glPopMatrix;
-import static org.lwjgl.opengl.GL11.glPushMatrix;
-import static org.lwjgl.opengl.GL11.glScaled;
-import static org.lwjgl.opengl.GL11.glTranslated;
+import static org.lwjgl.opengl.GL11.*;
```

Comment:
Please, no star imports. Revert this back to the way it was.

You can change the lines of imports before it gets converted to * in intelliJ. Set that to a high value, like 999.

Reply:
Hmm, IntelliJ did this automatically for me, so I thought it must be in line with the Terasology code style. I will get rid of them.